nautobot_circuit_maintenance/views.py

```diff
@@ -188,25 +192,25 @@ def get_maintenances_per_month(self):
 
         return self.queryset.count() / delta_months
 
-
-class CircuitMaintenanceListView(generic.ObjectListView):
-    """View for listing the config circuitmaintenance feature definition."""
-
+class CircuitMaintenanceUIViewSet(NautobotUIViewSet):
+    """Circuit Maintenance Class."""
     queryset = models.CircuitMaintenance.objects.order_by("-start_time")
-    table = tables.CircuitMaintenanceTable
-    filterset = filters.CircuitMaintenanceFilterSet
-    filterset_form = forms.CircuitMaintenanceFilterForm
+    table_class = tables.CircuitMaintenanceTable
+    filterset_class = filters.CircuitMaintenanceFilterSet
+    filterset_form_class = forms.CircuitMaintenanceFilterForm
     action_buttons = ("add", "export")
+    form_class = forms.CircuitMaintenanceForm
+    bulk_update_form_class = forms.CircuitMaintenanceBulkEditForm
 
-
-class CircuitMaintenanceView(generic.ObjectView):
-    """Detail view for specific circuit maintenances."""
-
-    queryset = models.CircuitMaintenance.objects.all()
-
-    def get_extra_context(self, request, instance):
+    def get_extra_context(self, request, instance): # pylint: disable=signature-differs
         """Extend content of detailed view for Circuit Maintenance."""
-        maintenance_note = models.Note.objects.filter(maintenance=instance)
+        if instance:
+            maintenance_note = Note.objects.filter(
+                assigned_object_id=instance.id,
+                assigned_object_type=ContentType.objects.get_for_model(models.CircuitMaintenance),
+            )
+        else:
+            maintenance_note = None
```

Comment:
It doesn't look like this is being used anymore so it can likely be removed

Reply:
@chadell Remove?